python/test/feature_extractor_test.py

```diff
@@ -232,6 +233,34 @@ def test_run_moment_fextractor(self):
         self.assertAlmostEqual(results[1]['Moment_feature_dis2nd_score'], 4696.668388042269, places=4)
         self.assertAlmostEqual(results[1]['Moment_feature_disvar_score'], 1121.519917231203, places=4)
 
+    def test_run_moment_fextractor_workfile_yuv_10b(self):
```

Comment:
For the two tests added, they require FFmpeg path to be configured. But in the CI loop, only tests without FFmpeg are run. You should move these two tests under test/extra, which can be run manually (they will be run in internal tests in vmaf_oss).

Reply:
Thanks, I didn't know this. By the way, `test/extra/ffmpeg_extratest.py` has some failing tests, maybe there were changes to VMAF and they weren't updated? 

```
======================================================================
FAIL: test_noref_moment_fextractor_with_noref_asset_notyuv_gaussianblur (__main__.NorefFeatureExtractorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/nilfm/vmaf/python/test/extra/ffmpeg_extratest.py", line 109, in tearDown
    super().tearDown()
  File "/Users/nilfm/vmaf/python/vmaf/tools/misc.py", line 516, in tearDown
    self.assertEqual([], self.verificationErrors)
AssertionError: Lists differ: [] != ['63.25902145061728 != 63.273976755401236 [227 chars]ce)']

Second list contains 3 additional elements.
First extra element 0:
'63.25902145061728 != 63.273976755401236 within 4 places (0.014955304783953238 difference)'

- []
+ ['63.25902145061728 != 63.273976755401236 within 4 places '
+  '(0.014955304783953238 difference)',
+  '5123.705637307099 != 5124.572131500771 within 4 places (0.8664941936722244 '
+  'difference)',
+  '1113.0346638689637 != 1111.996326793719 within 4 places (1.0383370752447263 '
+  'difference)']

======================================================================
FAIL: test_run_vmaf_fextractor_with_gaussian_blurring (__main__.ParallelFeatureExtractorTestNew)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/nilfm/vmaf/python/test/extra/ffmpeg_extratest.py", line 23, in tearDown
    super().tearDown()
  File "/Users/nilfm/vmaf/python/vmaf/tools/misc.py", line 516, in tearDown
    self.assertEqual([], self.verificationErrors)
AssertionError: Lists differ: [] != ['24.110899416666665 != 24.109545916666665[48 chars]ce)']

Second list contains 1 additional elements.
First extra element 0:
'24.110899416666665 != 24.109545916666665 within 4 places (0.0013535000000004516 difference)'

- []
+ ['24.110899416666665 != 24.109545916666665 within 4 places '
+  '(0.0013535000000004516 difference)']

======================================================================
FAIL: test_run_psnr_runner_with_notyuv_gblur (__main__.QualityRunnerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/nilfm/vmaf/python/test/extra/ffmpeg_extratest.py", line 164, in tearDown
    super().tearDown()
  File "/Users/nilfm/vmaf/python/vmaf/tools/misc.py", line 516, in tearDown
    self.assertEqual([], self.verificationErrors)
AssertionError: Lists differ: [] != ['50.99313338666667 != 51.12088967333333 w[44 chars]ce)']

Second list contains 1 additional elements.
First extra element 0:
'50.99313338666667 != 51.12088967333333 within 4 places (0.12775628666665995 difference)'

- []
+ ['50.99313338666667 != 51.12088967333333 within 4 places (0.12775628666665995 '
+  'difference)']

======================================================================
FAIL: test_run_vmaf_runner_with_notyuv_gblur (__main__.QualityRunnerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/nilfm/vmaf/python/test/extra/ffmpeg_extratest.py", line 164, in tearDown
    super().tearDown()
  File "/Users/nilfm/vmaf/python/vmaf/tools/misc.py", line 516, in tearDown
    self.assertEqual([], self.verificationErrors)
AssertionError: Lists differ: [] != ['10.039371696640373 != 77.293043887001 wi[41 chars]ce)']

Second list contains 1 additional elements.
First extra element 0:
'10.039371696640373 != 77.293043887001 within 0 places (67.25367219036063 difference)'

- []
+ ['10.039371696640373 != 77.293043887001 within 0 places (67.25367219036063 '
+  'difference)']

----------------------------------------------------------------------
Ran 7 tests in 5.389s

FAILED (failures=4)
```

Reply:
This is because these test values are run with internal tests, which uses a different ffmpeg build. That’s also why the open source CI loop doesn’t include these tests.